core/src/main/java/io/aiven/klaw/auth/KwAuthenticationSuccessHandler.java

```diff
@@ -62,25 +62,30 @@ public String getRedirectPage(HttpServletRequest request, Authentication authent
     String providerRoute = "{{ provider }}";
     String coralTopicsUri = "/coral/";
 
-    if (quickStartEnabled
-        && handleDbRequests
-            .getUsersInfo(
-                UtilMethods.getUserName(
-                    authentication.getPrincipal(), preferredUsernameAttribute, emailAttribute))
-            .getRole()
-            .equals(KwConstants.USER_ROLE)) {
-      return coralTopicsUri;
-    }
-
-    if (coralEnabled
-        && UtilControllerService.isCoralBuilt
-        && !handleDbRequests
-            .getUsersInfo(
-                UtilMethods.getUserName(
-                    authentication.getPrincipal(), preferredUsernameAttribute, emailAttribute))
-            .getRole()
-            .equals(KwConstants.SUPERADMIN_ROLE)) {
-      return coralTopicsUri;
+    try {
+      if (quickStartEnabled
+          && handleDbRequests
+              .getUsersInfo(
+                  UtilMethods.getUserName(
+                      authentication.getPrincipal(), preferredUsernameAttribute, emailAttribute))
+              .getRole()
+              .equals(KwConstants.USER_ROLE)) {
+        return coralTopicsUri;
+      }
+
+      if (coralEnabled
+          && UtilControllerService.isCoralBuilt
+          && !handleDbRequests
+              .getUsersInfo(
+                  UtilMethods.getUserName(
+                      authentication.getPrincipal(), preferredUsernameAttribute, emailAttribute))
+              .getRole()
+              .equals(KwConstants.SUPERADMIN_ROLE)) {
+        return coralTopicsUri;
+      }
+    } catch (Exception e) {
+      log.error("Could not retrieve user info");
```

Comment:
Since this not an issue that requires admin/operator intervention and is expected behavior, I suggest reducing the log level (e.g. not to trigger any log-based alerts for the error level).
```suggestion
      log.info("Could not retrieve user info");
```

Reply:
Makes sense. Updated.